easybuild/easyconfigs/p/PCRE2/PCRE2-10.31-foss-2018b.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'PCRE2'
+version = '10.31'
+
+homepage = 'http://www.pcre.org/'
+description = """
+ The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax
+ and semantics as Perl 5.
+"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'optarch': True, 'pic': True}
```

Comment:
@furstj `optarch` is enabled by default, so please change to:

```python
```

(we should make that change in all existing easyconfigs, I assume you just inherited this from an existing easyconfig...)